### ISSUE TITLE
The file 'download_dataset.sh' is missing. 

### ISSUE THREAD

**Issue by SeongwoongCho** [PremierTACO/premier-taco#1]
Hello, thank you for sharing the code of your inspirational work, PremierTACO. 
I am now trying to reproduce the results of PremierTACO, but I don't found '`download_dataset.sh`' in the github repository. 
Could you upload `download_dataset.sh` ?

**Comment by PremierTACO** [comment]
Hi, thanks for checking out our code and pointing out the missing data downloading file. We have now uploaded the ``download_dataset.sh`` script to the repo.

**Comment by SeongwoongCho** [comment]
@PremierTACO Thank you for sharing download code. 
I think there is a typo in download_dataset.sh, where for downloading dmc_eval_data, it is not `wget -P ${DATA_DIR} https://obj.umiacs.umd.edu/dmc-pretrain/acrobot_swingup_replay.zip`, but `wget -P ${DATA_DIR} https://obj.umiacs.umd.edu/dmc-pretrain/dmc_eval_data.zip`

And I also found that in `dmc_eval_data.zip`, some trajectories are missing (less than 20 trajectories). I list the number of trajectories in each subdirectory of dmc_eval_data as the following (directory_name, # of trajectories) : 
```
quadruped_run_expert20, 11
quadruped_run, 11
finger_spin_expert20, 11
finger_spin, 11
humanoid_walk_expert100, 52
humanoid_walk, 52
cup_catch_expert20, 20
cup_catch, 20
hopper_hop_expert20, 10
hopper_hop, 10
cheetah_run_expert20, 14
cheetah_run, 14
quadruped_walk_expert20, 21
quadruped_walk, 21
walker_walk_expert20, 20
walker_walk, 20
dog_trot_expert100, 100
dog_trot, 100 
```

Could you provide a dataset for evaluation that was used to produce results in the table 1?

**Comment by PremierTACO** [comment]
@SeongwoongCho Thanks for pointing this out. It looks like the evaluation dataset that we uploaded was contaminated for some tasks. We will take a close look at it and get back to you ASAP. Thanks again for letting us know this issue!

**Comment by PremierTACO** [comment]
@SeongwoongCho Thank you once more. We've re-collected the expert demonstration dataset using the same trained expert policy as before to create the evaluation dataset. We have tested the evaluation results against this new dataset, and they should align with what is reported in Table 1. The ``dmc_eval_dataset.zip`` is now updated, along with ``download_dataset.sh``.  Feel free to let us know if there are any additional questions.
